### PR TITLE
Bg/intake commands

### PR DESCRIPTION
**Pull request by BenG49**
Create Intake commands:

- IntakeAcquireCommand
- IntakeAcquireForeverCommand
- IntakeDeacquireCommand
- IntakeDeacquireForeverCommand
- IntakeExtendCommand
- IntakeRetractCommand

image
image

### CODE REVIEW

**Comment by selym3** [comment]
Fantastic pull request. Good work!

File: src/main/java/com/stuypulse/robot/commands/intake/IntakeAcquireForeverCommand.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+	
+	private final Intake intake;
+
+	public IntakeAcquireForeverCommand(Intake intake) {
+		this.intake = intake;
+		addRequirements(intake);
+	}
+
+	@Override
+	public void execute() {
```
Comment (Sam-Belliveau) [inline src/main/java/com/stuypulse/robot/commands/intake/IntakeAcquireForeverCommand.java:17]: this should be initialize

File: src/main/java/com/stuypulse/robot/commands/intake/IntakeExtendCommand.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	
+	private final Intake intake;
+
+	public IntakeExtendCommand(Intake intake) {
+		this.intake = intake;
+		addRequirements(intake);
+	}
+
+	@Override
+	public void execute() {
```
Comment (Sam-Belliveau) [inline src/main/java/com/stuypulse/robot/commands/intake/IntakeExtendCommand.java:17]: this should be initialize

**Review by Sam-Belliveau — CHANGES_REQUESTED** [review]

File: src/main/java/com/stuypulse/robot/commands/intake/IntakeExtendCommand.java (hunk 2 above)
Comment (selym3) [reply src/main/java/com/stuypulse/robot/commands/intake/IntakeExtendCommand.java:17]: agreed

File: src/main/java/com/stuypulse/robot/commands/intake/IntakeAcquireForeverCommand.java (hunk 1 above)
Comment (selym3) [reply src/main/java/com/stuypulse/robot/commands/intake/IntakeAcquireForeverCommand.java:17]: ^^^^

File: src/main/java/com/stuypulse/robot/commands/intake/IntakeRetractCommand.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+	private final Intake intake;
+
+	public IntakeRetractCommand(Intake intake) {
+		this.intake = intake;
+		addRequirements(intake);
+	}
+
+	@Override
+	public void execute() {
+		intake.retract();
```
Comment (selym3) [inline src/main/java/com/stuypulse/robot/commands/intake/IntakeRetractCommand.java:18]: initialize

**Review by Sam-Belliveau — APPROVED** [review]

**Pull request merged**